contracts/strategies/uranium/MasterUraniumStrategy.sol

```diff
@@ -0,0 +1,375 @@
+
+import "../../base/interface/IVault.sol";
+import "../../base/upgradability/BaseUpgradeableStrategy.sol";
+import "../../base/interface/pancakeswap/IPancakePair.sol";
+import "../../base/interface/pancakeswap/IPancakeRouter02.sol";
+
+contract MasterUraniumStrategy is BaseUpgradeableStrategy {
+    using SafeMath for uint256;
+    using SafeBEP20 for IBEP20;
+
+    address constant public pancakeswapRouterV2 = address(0x05fF2B0DB69458A0750badebc4f9e13aDd608C7F);
+    address constant public uraniumRouter = address(0xF4EE46Ac2BA83121F79c778ed0D950ffF11a18Ed);
+    address constant public u235 = address(0x28d2E3BB1Ec54A6eE8b3Bee612F03A85d3Ec0C0c);
+    address constant public u92 = address(0x670De9f45561a2D02f283248F65cbd26EAd861C8);
+
+    // additional storage slots (on top of BaseUpgradeableStrategy ones) are defined here
+    bytes32 internal constant _POOLID_SLOT = 0x3fd729bfa2e28b7806b03a6e014729f59477b530f995be4d51defc9dad94810b;
+    bytes32 internal constant _IS_LP_ASSET_SLOT = 0xc2f3dabf55b1bdda20d5cf5fcba9ba765dfc7c9dbaf28674ce46d43d60d58768;
+
+    // this would be reset on each upgrade
+    mapping (address => address[]) public pancakeswapRoutes;
+    address[] public U235_2rewardPath;
+
+    constructor() public BaseUpgradeableStrategy() {
+        assert(_POOLID_SLOT == bytes32(uint256(keccak256("eip1967.strategyStorage.poolId")) - 1));
+        assert(_IS_LP_ASSET_SLOT == bytes32(uint256(keccak256("eip1967.strategyStorage.isLpAsset")) - 1));
+    }
+
+    function initializeStrategy(
+        address _storage,
+        address _underlying,
+        address _vault,
+        address _rewardPool,
+        address _rewardToken,
+        uint256 _poolID,
+        bool _isLpToken
+    ) public initializer {
+
+        BaseUpgradeableStrategy.initialize(
+            _storage,
+            _underlying,
+            _vault,
+            _rewardPool,
+            _rewardToken,
+            80, // profit sharing numerator
+            1000, // profit sharing denominator
+            true, // sell
+            1e16, // sell floor
+            12 hours // implementation change delay
+        );
+
+        address _lpt;
+        (_lpt,,,,,,) = IUraniumMasterChefDepositFee(rewardPool()).poolInfo(_poolID);
+        require(_lpt == underlying(), "Pool Info does not match underlying");
+        _setPoolId(_poolID);
+
+        if (_isLpToken) {
+            address uniLPComponentToken0 = IPancakePair(underlying()).token0();
+            address uniLPComponentToken1 = IPancakePair(underlying()).token1();
+
+            // these would be required to be initialized separately by governance
+            pancakeswapRoutes[uniLPComponentToken0] = new address[](0);
+            pancakeswapRoutes[uniLPComponentToken1] = new address[](0);
+        } else {
+            pancakeswapRoutes[underlying()] = new address[](0);
+        }
+
+        setBoolean(_IS_LP_ASSET_SLOT, _isLpToken);
+
+        U235_2rewardPath = [u92, rewardToken()];
```

Comment:
I know I did not do this myself before, but it's better to define this in the ..Mainnet strategies instead of here